test/invoice_repo_test.rb

```diff
@@ -69,16 +68,15 @@ def test_create_a_new_invoice_instance
 
   def test_update_an_invoice
     time = Time.stubs(:now).returns(Time.parse("2011-10-10T14:48:00"))
-    # time.stubs(:day).returns(1)
     attributes = {status: :success,
                   updated_at: time
                   }
 
     invoice = @se.invoices.find_by_id(6)
     @se.invoices.update(6, attributes)
-
+    expected = Time.parse("2011-10-10T14:48:00 -0400")
```

Comment:
The missing piece to the puzzle! Yes this makes so much more sense now for being able to stub the time by setting our expected to the Time.parse value.